.gitlab-ci.yml

```diff
@@ -156,21 +183,34 @@ publish:s3:
     - export AWS_SECRET_ACCESS_KEY=$TMP_STORAGE_AWS_SECRET_ACCESS_KEY
     - apt update && apt install -yyq awscli
     # Fetch artifacts from temporary S3 bucket
-    - aws s3 cp s3://mender-gitlab-tmp-storage/$CI_PROJECT_NAME/$CI_PIPELINE_ID/deploy.tar.gz deploy.tar.gz
-    - tar xzf deploy.tar.gz
+    - aws s3 cp s3://mender-gitlab-tmp-storage/$CI_PROJECT_NAME/$CI_PIPELINE_ID/raspberrypi3.tar.gz raspberrypi3.tar.gz
+    - tar xzf raspberrypi3.tar.gz
```

Comment:
Missing fetch/extract for raspberry4 tarball.

Reply:
Good catch, fixing!